src/patternfly/components/Table/table-tree-view.scss

```diff
@@ -1,49 +1,77 @@
 $pf-c-tree-view--MaxDepth: 10;
 
 .pf-c-table {
+  --pf-c-table__tree-view-main--indent--base: calc(var(--pf-global--spacer--md) * 2 + var(--pf-c-table__tree-view-icon--MinWidth)); // based off of the expected width of the toggle button
+  --pf-c-table__tree-view-main--nested-indent--base: calc(var(--pf-c-table__tree-view-main--indent--base) - var(--pf-global--spacer--md)); // nested spacing that removes the toggle button's left padding, so the icon aligns with text on the node above it
+
   // main
-  --pf-c-table__tree-view-main--GridTemplateColumns: var(--pf-c-table--m-tree-view__toggle--Width) max-content 1fr;
-  --pf-c-table__tree-view-main--indent--base: var(--pf-global--spacer--lg);
-  --pf-c-table__tree-view-main--MarginLeft: calc(var(--pf-global--spacer--md) * -1); // based off of the expected width of the toggle button
+  --pf-c-table__tree-view-main--PaddingLeft: 0;
```

Comment:
I wonder if this should be `--pf-global--spacer--sm` > xl breakpoint so that the button icon aligns with the column header text

Current > 1200px
<img width="396" alt="Screen Shot 2021-03-01 at 1 51 53 PM" src="https://user-images.githubusercontent.com/35148959/109550841-6baabf00-7a95-11eb-93e4-4c964c18cf43.png">

< 1200px aligns since the table and button padding are 16px
<img width="309" alt="Screen Shot 2021-03-01 at 1 52 01 PM" src="https://user-images.githubusercontent.com/35148959/109550881-75342700-7a95-11eb-8f0b-7d3b4b99326b.png">